vllm/lora/request.py

```diff
@@ -0,0 +1,31 @@
+from dataclasses import dataclass
+
+
+@dataclass
+class LoRARequest:
```

Comment:
I have a question about the API design here. My initial inclination is to re-use the `lora_local_path` as the `lora_id`, and then `hash(lora_local_path)` as the `lora_int_id`. This leads to two questions in my mind:
 
1. Is there any reason not to do this?
2. If this would work fine, might it make sense for the library to do this internally so the user doesn't have to specify two extra parameters?

Reply:
I think that would work fine - I wanted to leave anything like that as a part of a followup integration with the OpenAI server after this PR is merged.

Reply:
Is integration with the OpenAI server in progress?  If not, I'm willing to help